code/modules/mob/living/carbon/human/species_types/wy_synths.dm

```diff
@@ -74,11 +73,8 @@
 	RegisterSignal(C, COMSIG_MOB_SAY, PROC_REF(handle_speech))
 	laws = new /datum/ai_laws/steward
 
+	C.physiology.force_multiplier *= 1.25 //We hit 25% harder with all weapons
 
-	var/obj/item/organ/appendix/A = C.getorganslot(ORGAN_SLOT_APPENDIX) // Easiest way to remove it.
-	if(A)
-		A.Remove(C)
-		QDEL_NULL(A)
```

Comment:
where this go?

Reply:
was never needed, powerhungry species don't have an appendix